document/src/main/java/com/ritense/document/domain/impl/request/NewDocumentRequest.java

```diff
@@ -19,13 +19,11 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.JsonNode;
-
-import com.ritense.document.domain.RelatedFile;
 import com.ritense.valtimo.contract.resource.Resource;
+
```

Comment:
You have removed a lot of empty lines in the imports. Should you not omit this one?